Utils/book_handler.rb

```diff
@@ -0,0 +1,73 @@
+# book_handler.rb
+require_relative '../Classes/item'
+require_relative '../Classes/book'
+require_relative '../Classes/label'
```

Comment:
- [ ] Just as I pointed out above about the incorrect casing, these `required classes` here are not imported correctly. the `c` in the classes should be small letters as that's how the folder was named. I'll suggest you always take note of these cases as your app will exhibit unpredictable behavior because of classes not being imported correctly. :+1: :wave: :smile: